appservice/src/TunnelProxy.ts

```diff
@@ -154,6 +160,9 @@ export class TunnelProxy {
         return new Promise<void>((resolve: () => void, reject: () => void): void => {
             const statusOptions: request.Options = {
                 uri: `https://${this._client.kuduHostName}/AppServiceTunnel/Tunnel.ashx?GetStatus`,
+                headers: {
+                    'User-Agent': 'VSCode'
```

Comment:
I thought about this a bit, but wasn't sure exactly what to do. This code is not actually in the extension but in a common library. This is being used by vscode-azureappservice and we have plans to use it in vscode-azurefunctions very soon. So including the actual extension name would have to be dynamic somehow.

I even looked up the spec :smile: https://tools.ietf.org/html/rfc7231#section-5.5.3

Given that, I think listing VSCode as a product makes sense, perhaps followed by a secondary product. By then I thought that I was overthinking it all and left it as you see it. Thoughts?

Reply:
Pragmatically, the app service guys asked me to add something so that they can filter their metrics easier. When we start work with vscode-azurefunctions that is another opportunity to change this if necessary.

Reply:
I'd suggest vscode-azuretools then for now.